src/services/configService.ts

```diff
@@ -184,82 +208,56 @@ export class ConfigService {
         || tenantId,
       appId: this.getOption(constants.variableKeys.appId)
         || process.env.AZURE_CLIENT_ID
-        || appId
+        || appId,
```

Comment:
extra comma

Reply:
That's actually how it's supposed to be. Most linters will complain if there's not a trailing comma. There are lots of places where I'm missing it (e.g. right above this one on line 188), but trailing comma is usually the ideal